tools/app_interface_reporter.py

```diff
@@ -270,7 +325,19 @@ def get_apps_data(date, month_delta=1, thread_pool_size=10):
                 else:
                     jobs_to_get[job['instance']].append(job)
 
-    job_history = get_build_history_pool(
+    # This is temporary, it can go into a secret or can come from
+    # queries.get_pipelines_providers()
+    token = secret_reader.read({
```

Comment:
This will be available in the cluster schema via the `automationToken` field once you find out what the tekton cluster is through graphql queries as suggested previously